libcontainer/cgroups/systemd/nosystemd.go

```diff
@@ -0,0 +1,22 @@
+// +build linux,no_systemd
+
+package systemd
+
+import (
+	"errors"
+
+	"github.com/opencontainers/runc/libcontainer/cgroups"
+	"github.com/opencontainers/runc/libcontainer/configs"
+)
+
+func IsRunningSystemd() bool {
+	return false
+}
+
+func NewUnifiedManager(config *configs.Cgroup, path string) (cgroups.Manager, error) {
+	return nil, errors.New("no lennartix")
```

Comment:
I think that the term used in this error message will be very confusing to the majority of users.

Reply:
fixed.